pom.xml

```diff
@@ -464,5 +464,33 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>update_version_in_files</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>io.github.floverfelt</groupId>
+						<artifactId>find-and-replace-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>exec</id>
```

Comment:
Est-ce que tu peux changer le nom de l'exécution en quelque chose comme "change version in README.md" ?